proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,232 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **Getting started**
+  - Introduction (current About/Kubeflow)
+  - Concepts (current Kubeflow Overview)
+  - Use Cases (current About/Use Cases)
+  - Install Kubeflow (current `Installing Kubeflow`, to redirect to `Deployment` homepage)
+  - [delete] Cloud Installation
+    
+- **Community**
+  - About (current About/Community)
+  - Contributing (current About/Contributing to Kubeflow + details on Working Groups)
+  - Documentation (merge current About/Docs + About/Style Guide for Kubeflow docs)
+  - Events (current About/Events Calendar)
+  - Google Summer of Code (current About/Google Summer of Code)
+    
+- **Components** (remove "of Kubeflow")
+  - Jupyter Notebooks (moved to under components)
```

Comment:
The name in the UI (at least 1.0) is "Notebook Servers", so we should be consistent in the documentation. And I know some people use it for Notebooks that aren't Jupyter so it's probably good to keep it more generic

Reply:
Hi @wdhorton this is a good point! 

@kubeflow/wg-notebook-leads WDYT?

Reply:
While we are 100% heading in the direction of natively supporting non-jupyter notebooks, this is currently a bit of a hack.

But I do think we should view the component (from a docs perspective) as "Notebook Servers" as that is how users see it. However, we will have to update the text of that page to reflect a less jupyter-centric view of the component.

Reply:
Would `Notebooks` (already dropping the uniqueness of Jupyter) or `Notebook Servers` make more sense from a user perspective?

Reply:
I've updated the doc to reflect this discussion!